.github/renovate.json

```diff
@@ -1,19 +0,0 @@
-{
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "branchPrefix": "users/renovate/",
-  "extends": [
-    "config:recommended",
-    ":maintainLockFilesMonthly",
-    ":automergePatch",
-    "schedule:automergeDaily"
-  ],
-  "git-submodules": {
-    "enabled": true
-  },
-  "packageRules": [
-    {
-      "matchPackageNames": ["third_party/googleapis"],
```

Comment:
One subtle change that is obscured by the rename is that this changed from `matchPackageNames` to `matchDepNames`.

https://github.com/renovatebot/renovate/discussions/33367 says they used to be equivalent but now depName is the correct one to use.

> ok, yes, so starting with 2024-08 packageName for git submodules no longer is the submodule path but the git repo URL. depName still contains the submodule path, so switching from matchPackageNames to matchDepNames makes things work as before.